src/store/gameReducer.js

```diff
@@ -0,0 +1,46 @@
+import { types } from './gameActions';
+
+export const initialState = {
+    status: "NOT_STARTED",
+    roundNum: 0,
+    currentPlayer: null, /* { username } */
+    players: [], /* [{ username, score }] */
+    currentCards: [], /* [{ cardId }] */
+    currentWord: '',
+    winner: null, /* { username } */
```

Comment:
comments left in?

Reply:
I think this is to remind us what the data structures are holding, also this file will be deleted in the next commit because I forgot to do it in this one